tests/serializers/test_player.py

```diff
@@ -0,0 +1,20 @@
+"""A module for testing player serializer."""
+import pytest
+from django.contrib.auth.models import User
+
+from api.models import Player
+from api.serializers import PlayerSerializer
+
+pytestmark = pytest.mark.django_db
+
+
+def test_player_serializer():
+    """Check if PlayerSerializer properly serializes the data identical to Player object."""
```

Comment:
Try not hardcoding class name in the docstrings - it's a pain to rename them later